GageStatsAgent/GageStatsAgent.cs

```diff
@@ -119,6 +119,8 @@ public class GageStatsAgent :DBAgentBase, IGageStatsAgent
         public GageStatsAgent(GageStatsDBContext context, IHttpContextAccessor httpContextAccessor) :base(context)
         {
             this._messages = httpContextAccessor.HttpContext.Items;
+            // not sure if we need this, NSSServices has it
```

Comment:
I think this is fine, since our requests are stateless. This change should make the request go faster because EF doesn't need to mark/track the entities.